test/server/cards/02_SHD/upgrades/RichReward.spec.ts

```diff
@@ -0,0 +1,95 @@
+describe('Rich Reward', function() {
+    integration(function(contextRef) {
+        describe('Rich Reward\'s Bounty ability', function() {
+            it('should give Experience token up to two units', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [
+                            { card: 'battlefield-marine', upgrades: ['rich-reward'] },
+                            { card: 'atst' },
```

Comment:
You can mix object and string forms in the same list. So this would be simpler as

`groundArena: [{ card: 'battlefield-marine', upgrades: ['rich-reward'] }, 'atst', 'yoda#old-master']`

Reply:
done